DetectorDescription/Core/test/data/assembly.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<DDDefinition>
+  <SolidSection>
+    <Assembly name="AssemblyVolume"/>
+    <TruncTubs name="ChildVolume" rMin="6.9551*m" rMax="9*m" cutAtStart="6.9551*m" cutAtDelta="7.20045*m" cutInside="true"
+	       startPhi="0*deg" deltaPhi="15*deg" zHalf="6.57005*m"/>
+  </SolidSection>
+  <LogicalPartSection>
+    <LogicalPart name="AssemblyVolume" category="unspecified">
+      <rSolid name="AssemblyVolume"/>
+      <rMaterial name="materials:Air"/>
+    </LogicalPart>
+    <LogicalPart name="ChildVolume" category="unspecified">
+      <rSolid name="ChildVolume"/>
+      <rMaterial name="materials:Air"/>
+    </LogicalPart>
+  </LogicalPartSection>
+  <PosPartSection label="">
+     <PosPart copyNumber="1">
+      <rParent name="AssemblyVolume"/>
+      <rChild name="ChildVolume"/>
+      <Translation z="17.4299*m" y="0.*m" x="0."/>
+    </PosPart>
+    <PosPart copyNumber="2">
+      <rParent name="AssemblyVolume"/>
+      <rChild name="ChildVolume"/>
+      <Translation z="-17.4299*m" y="0.*m" x="0."/>
+    </PosPart>
+    <PosPart copyNumber="13">
+      <rParent name="world:MotherOfAllBoxes"/>
+      <rChild name="AssemblyVolume"/>
+      <Translation z="0*m" y="0.*m" x="0."/>
+    </PosPart>
```

Comment:
Is it true that rotations can be defined for assemblies just like any other volume? I just noticed that there is no example of assembly rotation in this test xml.

Reply:
Yes, both `Assembly` and `ShapelessSolid` are defined as a "`Shape`" in DDL. A "`LogicalPart`" has to be defined for them to be used by "`PosPart`" or "`Algorithm`" or "`Division`", but the `Material` assigned to the `LogicalPart` is ignored.

Now that I think about it we ought to revert removal of the [DDL schema](https://cmssdt.cern.ch/lxr/source/DetectorDescription/Schema/DDLSchema.xsd?v=CMSSW_10_6_20) and use it for validating the XML files.

@smuzaffar - FYI. I think, we need it back to prevent diverging from the DDL. I used to us a commercial tool for validating them, but it has been a while ago. Perhaps, there is an open source one by now? I'll investigate it. Thanks!

Reply:
@ianna , the files are available via https://github.com/cms-data/DetectorDescription-Schema data external and one of these is already used by a unit test https://github.com/cms-sw/cmssw/blob/master/Geometry/CMSCommonData/test/run_DOMCount.py#L53